style.css

```diff
@@ -37,115 +41,120 @@ html {
 	padding: 15px;
 }
 
-.welcome-banner {
-	background-color: #EEEEEE;
-	height: 90vh;
-	
+@media screen and (max-width: 600px) {
+	.cv-button {
+	flex-direction: column;
+	font-size: 2rem;
+	}
 }
 
 .navigation-links {
 	color: #ffffff;
 	text-decoration: none;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
-    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #2866c9;
 }
 
-.hi-there-welcome-text {
-	display:flex;
-	align-items: center;
-	justify-content: center;
+@media screen and (max-width: 600px) {
+	.navigation-links {
+	font-size: 2rem;
+	}
+}
+
+/*navigation*/
```

Comment:
Rayna said its best to remove comments unless they are explaining a particularly difficult to understand section of code, plus I think the navigation section might have moved since this was written!